### PR TITLE
feat:adds small vertical and horizontal card

### PR DESCRIPTION
**Pull request by FarhanMobashir**
small-card
the preview is not working so please look at the attached image for reference

### CODE REVIEW

File: src/components/cards.css [hunk 1]
```diff
@@ -200,3 +200,101 @@
...
+  padding-bottom: var(--sp-sm);
+}
+
+/* pills */
+.pill {
+  font-size: var(--tx-14);
+  background-color: var(--green-100);
+  color: var(--green-500);
+  padding: 0.5rem 1rem;
+  border-radius: 50px;
```
Comment (RakshithBhat03) [inline src/components/cards.css:266]: If you want you can use a variable for this.

**Review by RakshithBhat03 — APPROVED** [review]
You can implement the change if you want to. Else, everything looks good. Code is well structured and has good use of variables. Good to merge

File: src/components/cards.css [hunk 2]
```diff
@@ -200,3 +200,101 @@
...
   margin: 0px 10px;
 }
+
+/* --------- small product card ------------ */
+.card-container-small {
+  width: 18.5rem;
+  box-shadow: 0px 0px 5px var(--black-200);
+  border-radius: var(--radius-md);
+  display: flex;
+  flex-direction: column;
```
Comment (nareshpoonia) [inline src/components/cards.css:210]: You can consider making a utility class of flex-col since you'll be using it often

**Review by nareshpoonia — APPROVED** [review]
Good work, just one comment, you can increase the use of utility classes

**Pull request merged**